extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -541,15 +541,28 @@ public static function getHostByDbName( $dbName ) {
 	 * @return string
 	 */
 	public static function getHostById( $cityId ) {
-		global $wgDevelEnvironment;
-
 		$hostName = \WikiFactory::getVarValueByName( 'wgServer', $cityId );
+		$hostName = \WikiFactory::fixURLForNonProductionEnv( $hostName );
+
+		return rtrim( $hostName, '/' );
+	}
+
+	/**
+	 * Calls WikiFactory::getLocalEnvURL only on non-production host names
+	 * @TODO MAIN-7994 remove this method when WikiFactory::getLocalEnvURL supports custom domains
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1135)